### DISCUSSION TITLE
Firefly wallet

### DISCUSSION THREAD

**Discussion by babi76** [iotaledger/firefly#7216] [Help & Support]
Dobrý deň mám problém s hry som si nahrala 1 008 734 i , samozrejme vybrať sa dalo iba to Firefly wallet a tu nastal problém iba preleteli peňaženkou a skončili v Mainnetu a nemôžem a ani neviem čo mám robiť keď že som nováčikom,viete mi poradiť?A stále nerozumiem ako sa to prepočítava vyhodilo mi to nejakých 130 € .tak že nemám vôbec nič .Prosím má niekto skúsenosti s takýmto problémom ale nejakú radu?

**Comment by ercankuruz** [comment]
I have wallet problem. It is giving belove code. What should I do 

??App Version: 1.7.3

platform: macOS
platformVersion: 11.5.2
platformArchitecture: x64
cpuCount: 4
totalMem: 8192.0 MB
freeMem: 173.3 MB

Error Type: [Preload Context] Error

Error: node-loader:
Error: dlopen(/Applications/Firefly.app/Contents/Resources/app.asar.unpacked/public/build/index.node, 1): Symbol not found: __ZNSt3__115basic_stringbufIcNS_11char_traitsIcEENS_9allocatorIcEEE7seekoffExNS_8ios_base7seekdirEj
  Referenced from: /Applications/Firefly.app/Contents/Resources/app.asar.unpacked/public/build/index.node (which was built for Mac OS X 13.0)
  Expected in: /usr/lib/libc++.1.dylib
 in /Applications/Firefly.app/Contents/Resources/app.asar.unpacked/public/build/index.node

**Reply by begonaalvarezd** [reply]
This issue is solved in the latest version, please upgrade to ^1.7.4

**Comment by babi76** [comment]
Problém vyriešený ale moje finančné prostriedky som stále nevidela a nikomu sa nerači mi ich poslať (vrátiť) do novej peňaženky.

**Reply by begonaalvarezd** [reply]
Im sorry but I dont understand what is exactly your issue, could you please give us some details on what exactly is the issue you are experiencing? Thanks

**Comment by ercankuruz** [comment]
Thanks for your interest,

There was login problem, but I’ve sorted it out. 

Best Regards


Best Regards,
 
Huseyin Ercan KURUZ | Certifying Engineer
m. +971 (0)58 891 1633
***@***.*** | flydubai.com

**Comment by ercankuruz** [comment]
Hi,

Hi, I've updated my firefly. After new update I've lost all my SMR and Assambly coins in my wallet. How can I get them back
Best Regards,
 
Huseyin Ercan KURUZ | Certifying Engineer
m. +971 (0)58 891 1633
***@***.*** | flydubai.com

**Reply by begonaalvarezd** [reply]
Quick information about ASMB and SMR:

- Assembly: please read https://blog.iota.org/stardust-upgrade-iota-tokenomics/. The assembly project does not exist anymore, but your rewards got airdropped to you in form of IOTAs. They will appear in the new version 2.0.3 in the vesting tab. If you are missing balances you can run the balance finder from settings > advanced > balance finder, or if you are missing rewards you can also run the rewards finder from the vesting tab, the '....' menu, search vesting rewards
- Shimmer: what the old firefly version displayed was just a representation of what you owned, but you couldnt control the shimmers from the iota app because your rewards are in the shimmer network, not the iota network. You must download the firefly shimmer app to control those funds, you can read more about it here https://blog.shimmer.network/how-to-claim-smr-firefly/

**Comment by babi76** [comment]
Nerozumiem tomu preco teraz záujem o to aby ste mi pomohli lebo čo vlastne ide ja som to s firefly vzdala hneď po pokuse keď som požiadala o pomoc a keď som kontaktovala platformou ioty tak záhadou sa prerušilo spojenie tak že s podvodníkmi mám svoje skúsenosti toto boli malé drobaky okolo 150$ ale keď vás úmyselne odberu o niekoľko miliónov € a to úmyselne to už je na zamyslenie ale dala som im ultimátum dokedy majú danú situáciu vyriešiť inak podám trestné oznámenie pre kiberkriminaliru lebo sa to množi ako vši inak ak peňaženka nespĺňa požiadavky zrušiť a nájsť si novú,ktrora vám bude vyhovovať najlepšie mať niekoľko a skúšať ktorá vám lepšie vyhovuje.
     To je jediné čo vám môžem poradiť.
   Github je pre mňa Španielska dedina tak tam nechodím ,ale občas mi ju vyhodí ako možnosť ale ja to ignorujeme.
   Dajte vedieť ako ste dopadli ja som momentálne na nule ale stále sa snažím dúfať že sa to vyrieši .
Prajem príjemný deň 

> ** Mon, 04 Dec 2023 01:55:24 -0800 - "iotaledger/firefly" ***@***.***>, "iotaledger/firefly" ***@***.***> **
>
> Hi,
> 
> Hi, I've updated my firefly. After new update I've lost all my SMR and Assambly coins in my wallet. How can I get them back
> Best Regards,
>  
> Huseyin Ercan KURUZ | Certifying Engineer
> m. +971 (0)58 891 1633
> ***@***.*** | flydubai.com
>
